src/do_command.c

```diff
@@ -511,6 +511,8 @@ static int child_process(entry * e, char **jobenv) {
 					putc(ch, mail);
 #if defined(SYSLOG)
 				if (SyslogOutput) {
+					if (ch == '\r')
```

Comment:
Could you please rather move the condition (negated) to the if (SyslogOutput) ?

Reply:
Updated the commit.